content/en/examples/service/networking/minimal-ingress.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     nginx.ingress.kubernetes.io/rewrite-target: /
 spec:
+  ingressClassName: nginx
```

Comment:
We should do this (or something like it). I wonder if we should, for completeness, also add a second sample manifest for that IngressClass? If we do so, that IngressClass should set the controller name to `k8s.io/ingress-nginx`.


How about using a different name here? For example, `nginx-example`?

Reply:
Definitely, thanks, see also my comment in the PR about that.